packages/video_player/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.10.2
+
+* Adds optional VideoFormat used to signal what format the plugin should try.
```

Comment:
Note it is only on Android

Reply:
The file was updated already; outdatd comment?